From 6b290e64ee2bbe7974ee43ce49297ee9496f18d6 Mon Sep 17 00:00:00 2001
From: kargl <kargl@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Fri, 8 Apr 2011 21:41:35 +0000
Subject: [PATCH] 2011-04-08  Steven G. Kargl  <kargl@gcc.gnu.org>

        PR target/47829
        gcc/config.gcc: disable unwind table generation for crtbegin/crtend
        on i386-*-freebsd.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@172214 138bc75d-0d04-0410-961f-82ee72b054a4

index 58f6787..1f13564 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -1212,6 +1212,7 @@ x86_64-*-elf*)
 	;;
 i[34567]86-*-freebsd*)
 	tm_file="${tm_file} i386/unix.h i386/att.h dbxelf.h elfos.h ${fbsd_tm_file} i386/freebsd.h"
+	tmake_file="${tmake_file} i386/t-crtstuff"
 	;;
 x86_64-*-freebsd*)
 	tm_file="${tm_file} i386/unix.h i386/att.h dbxelf.h elfos.h ${fbsd_tm_file} i386/x86-64.h i386/freebsd.h i386/freebsd64.h"
-- 
1.7.0.4

